content/docs/manage/projects.md

```diff
@@ -108,6 +109,19 @@ For more information about available plan limits, see [Neon plans](/docs/introdu
 The Storage page also outlines Neon **Archive storage**** policy, if applicable to your Neon plan. For more information on this topic, see [Branch archiving](/docs/guides/branch-archiving).
 </Admonition>
 
+### Schedule updates for your project
+
+To keep your Postgres instances up to date, Neon applies updates that include Postgres minor version upgrades, security patches, and new features. Updates are applied to the computes within your project. They require a quick compute restart and typically occur weekly for active computes or those used within the last 30 days.
```

Comment:
I personally would clarify how 'quick' the update is, e.g. "a few seconds" or "up to X seconds"

Reply:
done!